app/org/sagebionetworks/bridge/cache/CacheProvider.java

```diff
@@ -129,14 +168,16 @@ public void removeString(String cacheKey) {
         try {
             jedisOps.del(cacheKey);
         } catch(Throwable e) {
-            promptToStartRedisIfLocalEnv(e);
+            promptToStartRedisIfLocal(e);
             throw new BridgeServiceException(e);
         }
     }
 
-    private void promptToStartRedisIfLocalEnv(Throwable e) {
-        if (BridgeConfigFactory.getConfig().isLocal()) {
-            throw new BridgeServiceException("Cannot find cache service, have you started a Redis server? (original message: "+e.getMessage()+")");
+    private void promptToStartRedisIfLocal(Throwable e) {
+        if (e.getClass() == JedisException.class && BridgeConfigFactory.getConfig().isLocal()) {
```

Comment:
The code being replaced catches Throwable.  I'm limiting it to JedisException.  Still not ideal but should be less noisy than the original code.